locales-src/asrouter.ftl

```diff
@@ -81,3 +81,25 @@ cfr-whatsnew-button =
   .tooltiptext = What’s New
 
 cfr-whatsnew-panel-header = What’s New
+
+## Bookmark Sync
+
+cfr-sync-bookmarks-type = Recommended Feature
+cfr-sync-bookmarks-header = Get this bookmark on your phone
+cfr-sync-bookmarks-body = Take your bookmarks, passwords, history and more everywhere you're signed into Firefox.
+cfr-sync-bookmarks-cta-button = Turn on Sync
+
+## Send Tab
+
+cfr-send-tab-type = Recommended Feature
+cfr-send-tab-header-1 = Read this on the go
```

Comment:
I don't see this mentioned in the bug?
https://bugzilla.mozilla.org/show_bug.cgi?id=1569696

Reply:
This one is from https://bugzilla.mozilla.org/show_bug.cgi?id=1569692; the other strings overlap (send tab) so I put them together.

Reply:
Thanks. That bug wasn't mentioned in https://bugzilla.mozilla.org/show_bug.cgi?id=1571023, had no idea there was a fifth one